.github/workflows/template-janitor.yml

```
@@ -0,0 +1,75 @@
+# This workflow should cleanup everything unneeded from the template project
+
+name: Template Janitor
+on:
+  push:
+    branches:
+      - main # maybe all to run "tests" on every change?
+
+env:
+  TEMPLATES_PATH: ".github/template"
+  
+jobs:
+
+  template-cleanup:
+    name: Cleanup after create
+    runs-on: ubuntu-latest
+    steps:
+      - name: Fetch Sources
+        uses: actions/checkout@v2.4.0
+
+      - name: Get organization and project name 
+        run: |
+          echo "NEW_ORG=${{ github.actor }}" >> $GITHUB_ENV
+          echo "NEW_PROJECT=${{ github.event.repository.name }}" >> $GITHUB_ENV
+
+      - uses: octokit/request-action@v2.x
+        id: get_repo_meta
+        with:
+          route: GET /repos/{owner}/{repo}
+          owner: ${{ env.NEW_ORG }}
+          repo: ${{ env.NEW_PROJECT }}
+        env:
+          GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+      
+      # Rename all cpp_starter_project occurences to current repository and remove this workflow
+      - name: Insert new org and project
+        run: |
+          # rename the CMake project to match the github project
+          sed -i "s/myproject/${{ github.event.repository.name }}/g" CMakeLists.txt
```

```suggestion
          sed -i "s/myproject/${{ github.event.repository.name }}/gi" CMakeLists.txt configured_files/config.hpp.in
```

To catch the names in the config.hpp.in example

---

I'll add this as soon as i can

---

You can just hit the "commit suggestion" button if you think it would work

---

I know. But not from the Android Github App 😅

---

Thanks for the suggestion!